RgsToElectrumServerMiddleware/Program.fs

```diff
@@ -0,0 +1,44 @@
+open System
+open System.Threading.Tasks
+open Microsoft.AspNetCore.Builder
+open Microsoft.Extensions.Hosting
+
+open GWallet.Backend
+open GWallet.Backend.UtxoCoin
+
+[<EntryPoint>]
+let main args =
+    let builder = WebApplication.CreateBuilder(args)
+    let app = builder.Build()
+
+    app.Urls.Add("http://0.0.0.0:5108");
```

Comment:
@aarani nit, no need for parens and no need for semicolon

Reply:
:5108 hardcoded here, but is also present in launchSettings.json, DRY?

Reply:
The most important point that you need to keep in mind is this launchSettings. json file is only used within the local development machine

Reply:
so in the local development machine, the line `app.Urls.Add("http://0.0.0.0:5108")` is not executed?

Reply:
launchSettings.json was unused and removed.